ts/features/it-wallet/navigation/ItwStackNavigator.tsx

```diff
@@ -47,9 +56,5 @@ export const ItwStackNavigator = () => (
       name={ITW_ROUTES.ACTIVATION.CIE_WRONG_PIN_SCREEN}
       component={CieWrongCiePinScreen}
     />
-    <Stack.Screen
-      name={ITW_ROUTES.ACTIVATION.TYP}
-      component={ItwActivationTypScreen}
-    />
```

Comment:
why is it removed?

Reply:
This has been removed because the "final" thank you page is already contained in the `PidActivationScreen.tsx`. 
The loading screen after the CIE login flow is instead contained in the `PidPreviewScreen.tsx`. 
What I'm trying to achieve here is that an attached saga in `PidActivationScreen.tsx` will decode the PID, hence the loading screen. If everything goes well then it will render its claims.